Firestore/core/test/firebase/firestore/util/executor_std_test.cc

```diff
@@ -212,11 +211,16 @@ TEST_F(ScheduleTest, PopBlockingIsNotAffectedByIrrelevantRemovals) {
     EXPECT_EQ(schedule.PopBlocking(), 1);
   });
 
-  while (schedule.empty()) {
+  // Wait (with timeout) for both values to appear in the schedule.
+  while (schedule.size() != 2) {
```

Comment:
This might eventually benefit from refactoring. `wait(timeout, [&]->bool { schedule.size() != 2 });` or similar.

No action required.